hydrolib/core/dflowfm/structure/models.py

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
This class appears to still make use of from enum import Enum.
mainly because of the DambreakAlgorithm(int, Enum).
Should this also be updated to IntEnum as similar to the StrEnum, or does this not cause any issues?

Reply:
Only the str, Enum mixins were giving problems, so we should only update these.